recipes/Time_Series/Bike_Sharing_Finetuning_with_Exogenous.ipynb

```diff
@@ -0,0 +1,1658 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Transportation Demand Forecasting with Granite Time Series - Zero-shot Inference and Fine-tuning with Exogenous Inputs\n",
+    "\n",
+    "TinyTimeMixers (TTMs) are compact pre-trained models for Multivariate Time-Series Forecasting, open-sourced by IBM Research. With less than 1 Million parameters, TTM introduces the notion of the first-ever \"tiny\" pre-trained models for Time-Series Forecasting. TTM outperforms several popular benchmarks demanding billions of parameters in zero-shot and few-shot forecasting and can easily be fine-tuned for multivariate forecasts.\n",
+    "\n",
+    "In this recipe, we cover zero-shot forecasting, as well as fine-tuning. This example makes use of the Kaggle bike sharing [dataset](https://www.kaggle.com/datasets/lakshmi25npathi/bike-sharing-dataset) which contains bikes rental demand with weather and seasonal information."
```

Comment:
we're always catching heck about data sets even when we're not hosting them. Is this Kaggle one OK?

Reply:
These were approved for use back when we used this data for Think.

Reply:
That's cool, it looks like an OK data set to me, but people have been super protective (then someone will do an illegal twitter pull and try to host it in our repo- suffice to say I can't track any thread of rhyme or reason.